react/19/replace-default-props/src/index.ts

```diff
@@ -8,10 +8,8 @@ import type {
   ObjectProperty,
 } from "jscodeshift";
 
-import {
-  getFunctionComponents,
-  getFunctionName,
-} from "@codemod.com/codemod-utils";
+import { getFunctionName } from "@codemod.com/codemod-utils/src/jscodeshift/function.js";
+import { getFunctionComponents } from "@codemod.com/codemod-utils/src/jscodeshift/react.js";
```

Comment:
Aren't these functions exported by the main package file?

Reply:
i think they are has some problems, but i don't remember correctly, but i remember it has error.